### ISSUE TITLE
Feature request: Keyboard Shortcuts in Help Pane

### ISSUE THREAD

**Issue by omnomz** [rstudio/rstudio#5149]
Hi guys, just two small suggestions:

1) **Keyboard shortcuts to navigate Previous topic/Next topic in Help pane**
I rely on Help quite a bit as a newbie- so it's extremely helpful to be able to search for help within RStudio instead of having to switch to a browser. However, it is a tad inconvenient to have to locate the tiny blue arrows to navigate back and forth though topics.  

2) **Tab navigation shortcuts to work in other panes**
Currently they only work in Source even when it's not in focus/minimised. 

Thank you!

**Comment by gtritchie** [comment]
There are existing shortcuts for many tabs under View menu but I plan to add next/prev tab commands that aren’t limited to source tabs as part of accessibility work.

**Comment by omnomz** [comment]
That's great to hear!

**Comment by francisbarton** [comment]
I was going to open a new feature request issue but thought it might be better to add on to this one if that's ok. Let me know if not - and I will go file a new issue.

My request is to be able to search in the Help pane just with the keyboard:

If I do `Ctrl`+`3` to focus the Help pane, then `Ctrl`+`F` the cursor focus moves to the "Find in topic" text box. But I can't yet find a shortcut to move the focus to the general Help search box (at the top right of the pane).
My suggestion would be that `Ctrl`+`3` followed by `/` should put the focus there, so the user can just type and submit the search query without needing a mouse.

(PS yes I know `Ctrl`+`2` followed by `?$query` does this but it feels like you should be able to search from the Help pane itself.)
